pytorch_lightning/callbacks/quantization.py

```diff
@@ -174,7 +182,15 @@ def _check_feasible_fuse(self, model):
 
     def on_fit_start(self, trainer, pl_module):
         # QuantStub converts tensors from floating point to quantized
-        pl_module.quant = torch.quantization.QuantStub()
+        if self.max_num_inputs is None:
+            if pl_module.example_input_array is None:
+                raise MisconfigurationException(
+                    '`max_num_inputs` or `model.example_input_array must be provided to determine number of QuantStubs'
+                )
+            num_inps = len(pl_module.example_input_array)
+        else:
+            num_inps = self.max_num_inputs
+        pl_module.quants = nn.ModuleList([torch.quantization.QuantStub() for _ in range(num_inps)])
```

Comment:
This means it will fail for previous users who don't update right?

Maybe the previous behaviour should be kept if none are provided

Reply:
Good point, I'll update so that a single `QuantStub` is used if neither `max_num_inputs` or `example_input_array` is provided. Then there won't be any breaking changes. Maybe a user warning would be good though? The traceback for multi-input models is not super intuitive

Reply:
> Maybe a user warning would be good though?

Sure